exercises/concept/cars-assemble/.docs/introduction.md

```diff
@@ -15,6 +15,20 @@ For the sake of this exercise you will only be dealing with:
 
 Go supports the standard set of arithmetic operators of `+`, `-`, `*`, `/` and `%` (remainder not modulo).
 
+## Arithmetic Operators
+
+Go supports many standard arithmetic operators:
+
+| Operator | Example        |
+| -------- | -------------- |
+| `+`      | `4 + 6 == 10`  |
+| `-`      | `15 - 10 == 5` |
+| `*`      | `2 * 3 == 6`   |
+| `/`      | `13 / 3 == 4`  |
+| `%`      | `13 % 3 == 1`  |
+
+For integer division, the remainder is dropped (eg. `5 / 2 == 2`).
+
```

Comment:
As said before this is the only file the student really sees so in the spirit of "introducing the concept" I would add one sentence like this even if it currently not part of the exercise:
"Go also has shorthand assignment operators for the operators above, e.g. `a += 5` is short for `a = a + 5`, and it support the increment and decrement operators `++` and `--`."